application/app/Http/Controllers/TaskController.php

```diff
@@ -78,13 +79,21 @@ public function index(Request $request, Project $project)
      *
      * @return \Illuminate\Http\Response
      */
-    public function create(Project $project)
+    public function create(Request $request, Project $project)
     {
         $task_kinds = TaskKind::all();
         $task_statuses = TaskStatus::all();
         $task_categories = TaskCategory::all();
         $assigners = User::all();
 
+        // 遷移元がtasks.create以外ならtmp_filesのセッションを破棄する処理
+        //（投稿画像の一時保存及び破棄時、store失敗時、更新ボタン押下時に一時保存画像を残すことを想定した）
```

Comment:
細かいですが、スタイルをきちんとあわせれると良いです
`//`の後にスペースがあったり、なかったりするようなので

```suggestion
        // （投稿画像の一時保存及び破棄時、store失敗時、更新ボタン押下時に一時保存画像を残すことを想定した）
```